manifests/config/apache.pp

```diff
@@ -96,7 +96,7 @@
   Pattern['^(https?|unix)://'] $proxy_backend = 'unix:///run/foreman.sock',
   Boolean $proxy_add_headers = true,
   Hash $proxy_params = { 'retry' => '0' },
-  Array[String] $proxy_no_proxy_uris = ['/pulp', '/pulp2', '/streamer', '/pub', '/icons'],
+  Array[String] $proxy_no_proxy_uris = ['/pulp', '/pulp2', '/streamer', '/pub', '/icons', '/server-status'],
```

Comment:
Are `/pulp2` and `/streamer` still things we need to proxy?

Reply:
Streaming is handled directly through the standard API in Pulp 3 now. For `/pulp2` I think this was used during transition and I cannot find it in use anymore.

Reply:
Let's get that in as-is, and drop pulp/stream stuff in a different PR?

Reply:
Sounds good. We'll have a major version bump coming up anyway.

Reply:
Exactly my thoughts, yeah!